easybuild/easyconfigs/i/ImageMagick/ImageMagick-7.0.11-14-GCCcore-10.3.0.eb

```
@@ -25,6 +25,8 @@ dependencies = [
     ('LibTIFF', '4.2.0'),
     ('LittleCMS', '2.12'),
     ('Pango', '1.48.5'),
+    ('pixman', 0.40.0'),
```

```suggestion
    ('pixman', '0.40.0'),
```
Ditto throughout @bedroge